src/Castle.Core/DynamicProxy/Contributors/InterfaceMembersWithDefaultImplementationCollector.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright 2004-2022 Castle Project - http://www.castleproject.org/
```

Comment:
Should be `2004-2023`, not `2022`. (I cherry-picked this code from an older code draft of mine.)